packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -287,7 +287,11 @@ export function getValidatorApi({
     // as of now fee recipient checks can not be performed because builder does not return bid recipient
     {
       skipHeadChecksAndUpdate,
-    }: Omit<routes.validator.ExtraProduceBlockOps, "builderSelection"> & {skipHeadChecksAndUpdate?: boolean} = {}
+      commonBlockBody,
+    }: Omit<routes.validator.ExtraProduceBlockOps, "builderSelection"> & {
```

Comment:
I don't see the usage of this type `Omit<routes.validator.ExtraProduceBlockOps, "builderSelection">`, can we drop it?

Reply:
Somehow it's used on the line 400 of this file. I will clean this up in a separate PR in-progress to clean up this flow.